newIDE/app/src/UI/Alert/ConfirmDeleteDialog.js

```diff
@@ -23,21 +25,43 @@ type Props = {|
 |};
 
 function ConfirmDeleteDialog(props: Props) {
+  const { open, confirmText } = props;
   const [textInput, setTextInput] = React.useState<string>('');
+  const confirmButtonRef = React.useRef<?ButtonInterface>(null);
   const canConfirm = props.confirmText ? textInput === props.confirmText : true;
+  const shouldAutofocus = useShouldAutofocusInput();
 
   const onConfirm = () => {
     if (!canConfirm) return;
     props.onConfirm();
     setTextInput('');
   };
 
+  React.useEffect(
+    () => {
+      if (open && shouldAutofocus && !confirmText) {
+        // If the dialog is opened and autofocus should be set and there is no confirm text
+        // to enter, focus Confirm button to enable quick deletion with only keyboard navigation.
+        setTimeout(
+          () => {
+            if (confirmButtonRef.current) {
+              confirmButtonRef.current.focus();
+            }
+          },
+          // Wait for component to be mounted so that confirmButtonRef targets something.
```

Comment:
Give a try to `useLayoutEffect` without setTimemout

Reply:
It does not work unfortunately